### ISSUE TITLE
Detecting Smishing messages

### ISSUE THREAD

**Issue by mderosa001** [mderosa001/Capstone_Issues#6]
### Title
Detecting Smishing Messages Using BERT and Advanced NLP Techniques
### URL
https://dl.acm.org/doi/10.1007/s42979-024-03532-7
### Summary 
Another dive into the use of BERT in spam detection. specifically in SMS messaging. 
### Citation
@article{10.1007/s42979-024-03532-7,
author = {Jain, Ankit Kumar and Kaur, Kamaljeet and Gupta, Naveen Kumar and Khare, Ankit},
title = {Detecting Smishing Messages Using BERT and Advanced NLP Techniques},
year = {2025},
issue_date = {Feb 2025},
publisher = {Springer-Verlag},
address = {Berlin, Heidelberg},
volume = {6},
number = {2},
url = {https://doi.org/10.1007/s42979-024-03532-7},
doi = {10.1007/s42979-024-03532-7},
abstract = {The study focuses on the detection of smishing attacks, a variant of phishing that exploits text messages to dupe mobile users into revealing private information, downloading malware, or clicking malicious links. Despite the existence of various approaches, there remains a gap in timely smishing detection and the ability to effectively distinguish between smishing and legitimate SMS messages. To address these issues our approach focuses on detecting smishing by identifying key features and using advanced models like BERT, combined with different loss functions. Through an exploration of embeddings including TF-IDF and BERT, we developed a classifier model that achieved satisfactory accuracy. Notably, BERT emerged as the most effective, adept at capturing contextual nuances and word associations. To further enhance accuracy, we transformed the original tri-class challenge into dual binary classification tasks, employing two multi-layer neural networks. Our research highlights the application of advanced Natural Language Processing (NLP) techniques such as tokenization and self-attention mechanisms for precise message categorization into smishing, ham, or spam groups. The logistic regression classifier achieved the highest accuracy of 94\% on the 4:1 ratio dataset. In summary, our approach enhances security by accurately detecting smishing through a combination of feature extraction, advanced NLP techniques, and effective classification models.},
journal = {SN Comput. Sci.},
month = jan,
numpages = {15},
keywords = {Smishing, Phishing, SMS, Malware, URL, BERT}
}
